yt_dlp/extractor/twitch.py

```diff
@@ -525,6 +528,82 @@ def _extract_storyboard(self, item_id, storyboard_json_url, duration):
                 } for path in images],
             }
 
+    def _extract_chat(self, vod_id):
+        chat_history = []
+        has_more_pages = True
+        retry_sleep = 5
+        max_retries = 3
+        retries = 0
+        pagenum = 1
+        gql_ops = [
+            {
+                'operationName': 'VideoCommentsByOffsetOrCursor',
+                'variables': {
+                    'videoID': vod_id,
+                    # 'cursor': <filled in in subsequent requests>
+                }
+            }
+        ]
```

Comment:
```suggestion
        gql_ops = [{
            'operationName': 'VideoCommentsByOffsetOrCursor',
             'variables': {'videoID': vod_id},
        }]
```

Reply:
do you think I could somehow keep the comment about the `cursor`? I think it helps in understanding what will be sent.
I had an attempt at it, let me know if you dont like it